doc/source/api/index.rst

```diff
@@ -124,6 +124,7 @@ API
        PHP <http://dl.maptools.org/dl/php_ogr/php_ogr_documentation.html>
        R <https://cran.r-project.org/web/packages/gdalraster/index.html>
        Rust <https://github.com/georust/gdal>
+       Ruby <https://github.com/telus-agcg/ffi-gdal>
```

Comment:
Please move it above Rust to keep lexicographic order

Reply:
@rouault , done!